docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -85,16 +85,15 @@ Now you are ready to write code that reads data from and writes data to Blob sto
 This example shows how to create a container if it does not already exist:
 
     // Retrieve a reference to a container.
-    let container = blobClient.GetContainerReference("mycontainer")
+    let container = blobClient.GetContainerReference("photos")
```

Comment:
It feels to me like we should make each file run through "clean" end-to-end as a script (and probably provide a link to the script as well - oh for fully literate programming in this repo...)

However that means creating the right containers, local files etc., and sometimes resources will have been defined earlier in the script. That's a bit annoying but probably worth it?